cmpcat/a1a5cd04-cfcc-47da-841c-d0cf666fe417/component_category.lp

```diff
@@ -0,0 +1,10 @@
+(librepcb_component_category a1a5cd04-cfcc-47da-841c-d0cf666fe417
+ (name "AVR 8bits Microcontrollers")
```

Comment:
I think I would put the manufacturer into the category name (for those who don't know AVR):

```suggestion
 (name "Microchip AVR Microcontrollers")
```